internal/runnerinstall/k8s.go

```diff
@@ -238,7 +238,7 @@ func (i *K8sRunnerInstaller) InstallFlags(set *flag.Set) {
 	set.StringVar(&flag.StringVar{
 		Name:    "k8s-runner-image",
 		Target:  &i.Config.RunnerImage,
-		Default: installutil.DefaultRunnerImage,
+		Default: defaultRunnerImage,
```

Comment:
Should this be `defaultODRImage`? It looks like the website docs are showing this as the server image rather than the `-odr` prepended image.

Reply:
This matches what all the other platforms do, e.g.:
https://github.com/hashicorp/waypoint/blob/305d27d0242571dc04e9cfeb192439898dff070f/internal/runnerinstall/ecs.go#L255-L260

So, you might be right, but that would be a bug PR, maybe related to #3839 . This PR I'm just trying to make it consistent with what it was/currently is on other platforms.

Reply:
😖